mcux/mcux-sdk/devices/MIMX9596/drivers/ChangeLogKSDK.txt

```diff
@@ -0,0 +1,21 @@
+/*!
+@page driver_log Driver Change Log
```

Comment:
What is mcu-soc in the commit message. If this is another repo unrelated to zephyr can you please add a link to the repo?